CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+We love pull requests. Here's a quick guide.
+
+Fork, then clone the repo:
+
+    git clone git@github.com:your-username/school.git
+
+Set up your machine:
+
```

Comment:
I'd add "cd school" here. common mistake for beginners, esp if they are following a guide...